src/client/testing/utils/date-utility.ts

```diff
@@ -17,3 +19,32 @@ export const getDateInFuture = (
     year: futureDate.format('YYYY'),
   };
 };
+
+export const geRequiredtDateInFormat = (
+  numberOfDaysFromToday: number,
+  requiredformat: string,
+) => {
+  const date = new Date();
+  let requiredDate = '';
+  const futureDate = date.setDate(date.getDate() + numberOfDaysFromToday);
+  requiredDate = dayjs(futureDate).format(requiredformat);
+  return requiredDate;
+};
```

Comment:
This method could be a lot shorter and accept some default values, e.g. 
<img width="432" alt="image" src="https://github.com/user-attachments/assets/2567b67d-a67e-4b2a-8b77-17c7dda6ef10" />